de-lib/src/test/java/org/iplantc/de/apps/client/presenter/toolBar/AppsToolbarPresenterImplTest.java

```diff
@@ -197,17 +203,17 @@ protected void displayPublishError(Publishable result) {
 
     @Test public void verifyToolRequestDlgShown_onRequestToolSelected() {
         RequestToolSelected eventMock = mock(RequestToolSelected.class);
-        NewToolRequestDialog dlgMock = mock(NewToolRequestDialog.class);
-        when(requestToolDlgProviderMock.get()).thenReturn(dlgMock);
 
         /*** CALL METHOD UNDER TEST ***/
         uut.onRequestToolSelected(eventMock);
 
-        verify(requestToolDlgProviderMock).get();
-        verify(dlgMock).show(NewToolRequestFormView.Mode.NEWTOOL);
+        verify(newToolRequestDialogProviderMock).get(newToolDialogCaptor.capture());
+	newToolDialogCaptor.getValue().onSuccess(toolRequestMock);
```

Comment:
Looks like a weird spacing issue happened here

Reply:
Oh bah, that's probably a tabs-vs-spaces thing. I'll fix it before merge.